lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1810,3 +1810,170 @@ def assertChildren(self, block, child_usage_ids):
         Used to assert that sets of children are equivalent.
         """
         self.assertEquals(set(child_usage_ids), set(child.scope_ids.usage_id for child in block.get_children()))
+
+
+@override_settings(ANALYTICS_DATA_URL=True)
```

Comment:
It might be better to supply a string like http://api.test.com rather than a boolean to avoid confusing people about the expected type.